src/icon/iconview.js

```diff
@@ -22,36 +22,58 @@ export default class IconView extends View {
 	constructor() {
 		super();
 
-		const bind = this.bindTemplate;
+		/**
+		 * The inline SVG or in legacy mode the name of the icon
+		 * which corresponds with the name of the file in the
+		 * {@link module:theme/iconmanagermodel~IconManagerModel}.
+		 *
+		 * @observable
+		 * @member {String} #content
+		 */
+		this.set( 'content' );
 
 		this.template = new Template( {
 			tag: 'svg',
 			ns: 'http://www.w3.org/2000/svg',
 			attributes: {
-				class: [
-					'ck-icon'
-				]
-			},
-			children: [
-				{
-					tag: 'use',
-					ns: 'http://www.w3.org/2000/svg',
-					attributes: {
-						href: {
-							ns: 'http://www.w3.org/1999/xlink',
-							value: bind.to( 'name', name => `#ck-icon-${ name }` )
-						}
-					}
-				}
-			]
+				class: 'ck-icon',
+				viewBox: '0 0 20 20'
```

Comment:
The code on master works without it. I guess this is because `IconManagerModel` brings symbols which already have `viewBox="0 0 20 20"` set. We must figure out how to deal with it because there's no way we go with it like that.

We must either make it configurable via `IconView#viewBox` or find a solution without viewBox.